### DISCUSSION TITLE
How to change the highlight color?

### DISCUSSION THREAD

**Discussion by MrPandir** [nushell/nushell#12942] [Q&A]
My problem is that when selecting, I don’t see the text because it’s also white.
I would like to change the background highlight color to `#b5d5ff` for example.
I looked through the [documentation](https://www.nushell.sh/book/coloring_and_theming.html#coloring-and-theming-in-nu) and didn't find this, it probably can't be changed now.
<img width="47%"  align="left" alt="Screenshot without highlighting" src="https://github.com/nushell/nushell/assets/137798474/f6075e99-9816-4cf9-ab05-56b9723e2d50"><img width="47%" align="right" alt="Screenshot with highlighting" src="https://github.com/nushell/nushell/assets/137798474/6b6c6317-44d8-48b6-a3fb-daca899684ad">

**Comment by kaathewisegit** [comment]
I might be wrong, but I think that the selection highlighting is handled by the terminal, not the shell.  Typically, it inverts the colors: the background color becomes the foreground one, and visa versa.

**Reply by fdncred** [reply]
I could be wrong too but I think the colors are hard-coded in reedline. Someone would have to find the PR and look.

**Reply by MrPandir** [reply]
These are two different selections.

For example, I use the [rio terminal](https://raphamorim.io/rio/), it provides the ability to change this using a theme:
```toml
background = '#000000'
foreground = '#ffffff'
selection-background = '#b5d5ff'
selection-foreground = '#000000'
```
<img width="40%" src="https://github.com/nushell/nushell/assets/137798474/bf8c147f-6dfa-443b-993a-8606979fec23">


While nushell does not provide this and uses a **white** highlight, without the ability to change this, unfortunately.

**Reply by fdncred** [reply]
All it takes is someone to make a PR to change this. I'm pretty sure it would be accepted as part of our theming.
